python/ray/air/result.py

```diff
@@ -22,11 +24,21 @@ class Result:
         metrics: The final metrics as reported by an Trainable.
         checkpoint: The final checkpoint of the Trainable.
         error: The execution error of the Trainable run, if the trial finishes in error.
+        dataframe: The full result dataframe of the Trainable.
+            The dataframe is indexed by iterations and contains reported
+            metrics.
+        best_checkpoints: A list of tuples of the best checkpoints saved
+            by the Trainable and their associated metrics. The number of
```

Comment:
Nice! This is much cleaner than inferring. Thanks!
nit: the word `best_checkpoints` may be a bit misleading as be default all checkpoints are contained.